ACPs/118-warp-signature-request/README.md

```diff
@@ -0,0 +1,84 @@
+```text
+ACP: 118
+Title: Warp Signature Interface Standard
+Author(s): Cam Schultz (https://github.com/cam-schultz)
+Discussions-To: 
+Status: Proposed
+Track: Best Practices Track
+```
+
+## Abstract
+
+Proposes a standard Warp signature request AppRequest types so that Warp signatures may be requested in a VM-agnostic manner. To make this concrete, this standard type should be defined in AvalancheGo such that VMs can import it at the source code level. This will also simplify external signature aggregator implementations by allowing them to depend only on AvalancheGo for message construction, rather than individual VM codecs.
```

Comment:
Could you link the code definition of `AppRequest` here, and specify that the standard being proposed is for the format of the `request` bytes of an `AppRequest` that signify a request for a Warp signature?